runtime/src/main/java/io/micronaut/jackson/ObjectMapperFactory.java

```diff
@@ -80,6 +82,21 @@
     @Inject
     protected KeyDeserializer[] keyDeserializers = new KeyDeserializer[0];
 
+    /**
+     * Builds default Factory {@link JsonFactory} using properties from {@link JacksonConfiguration}.
+     *
+     * @param jacksonConfiguration The configuration
+     * @return The {@link JsonFactory}
+     */
+    @Requires(beans = JacksonConfiguration.class)
+    @Singleton
+    @BootstrapContextCompatible
+    public JsonFactory jsonFactory(JacksonConfiguration jacksonConfiguration) {
```

Comment:
does the `JsonFactory` not need to be wired into the `ObjectMapper`?

Reply:
It already is https://github.com/micronaut-projects/micronaut-core/blob/3.0.x/runtime/src/main/java/io/micronaut/jackson/ObjectMapperFactory.java#L95

Reply:
ok